.github/workflows/validate.yml

```diff
@@ -0,0 +1,68 @@
+name: validate
+on: [push, pull_request]
+jobs:
+  main:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        node: [10.14.2, 12, 14, 15]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v2
+
+      - name: ⎔ Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: 📥 Download deps
+        uses: bahmutov/npm-install@v1
+        with:
+          useLockFile: false
+
+      - name: ▶️ Run validate script
+        run: npm run validate
+
+      - name: ⬆️ Upload coverage report
+        uses: codecov/codecov-action@v1
+
+  release:
+    needs: main
+    runs-on: ubuntu-latest
+    if:
+      ${{ contains('master,beta,next,alpha', github.base_ref) &&
```

Comment:
This doesn't appear to work. This job should not run on this PR but it is... I'm not sure how else to skip this job for non-release branches 🤔

Reply:
It's not enough to do ?

```md
on:
    push:
        branches:
            - master
```

Reply:
Not 100%, but the docs say `github.base_ref` is only present on `pull_request` events. So it might be treating it like an empty string and it is matching the `contains` function. Just a possible guess to try something like `${{ github.base_ref != '' && contains(...` or `${{ contains(..., github.base_ref || 'not-found')`.

Reply:
Oh yeah! I think I need to use `ref` 👍 Thanks!